renderer/src/pages/Welcome.tsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { getFilAddress, getOnboardingCompleted } from '../lib/station-config'
+import { ReactComponent as StationLogoLight } from '../assets/img/station-logo-light.svg'
+
+const sleep = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms))
+
+const Welcome = (): JSX.Element => {
+  const navigate = useNavigate()
+
+  useEffect(() => {
+    (async () => {
+      await sleep(2000)
+      if (await getFilAddress()) {
+        return navigate('/dashboard', { replace: true })
```

Comment:
```suggestion
        return navigate('/dashboard')
```

I had a realization that we don't need any of these, since desktop apps usually have no concept of going back in time anyway (the user wouldn't be able to)

Reply:
Good catch. 
I'll open a follow-up PR to review it on other pages as well.